pkg/kubelet/cm/devicemanager/device_plugin_stub.go

```diff
@@ -104,6 +110,43 @@ func (m *Stub) Stop() error {
 	return m.cleanup()
 }
 
+// GetInfo is the RPC which return pluginInfo
+func (m *Stub) GetInfo(stream watcherapi.Registration_GetInfoServer) error {
```

Comment:
Can you update the design doc with this new stream model?
Not sure it's a good model to use a stream in order to send one message.

Do we really need to send registration successful couldn't this just be recorded in kubelet (current version)?
What kind of errors can be reported by the error field?

Reply:
error is a string. So whatever error message watcher has for the failure, it will be sent to plugin. S

Reply:
Right but do we need to send the watcher error to the device plugin?
The errors I can think of are:
- The plugin doesn't have a valid name
- fsnotify errors

In both case it seem to make more sense to just log these errors in the Kubelet rather than add another rpc call to the registration